esmvalcore/cmor/_fixes/cmip5/gfdl_cm2p1.py

```diff
@@ -56,6 +58,50 @@ def fix_data(self, cube):
         return cube
 
 
+class Sit(Fix):
+    """Fixes for sit"""
+
+    def fix_metadata(self, cubes):
+        """
+        Fix metadata.
+
+        Fixes bad bounds
+
+        Parameters
+        ----------
+        cube: iris.cube.Cube
+
+        Returns
+        -------
+        iris.cube.Cube
+
+        """
+        cube = self.get_cube_from_list(cubes)
+        time = cube.coord('time')
+        if self._fix_required(time):
+            new_bounds = np.empty(time.bounds.shape, time.bounds.dtype)
+            for i, point in enumerate(time):
+                date = point.units.num2date(point.points[0])
+                start = cftime.DatetimeJulian(date.year, date.month, 1)
+                if date.month < 12:
+                    end = cftime.DatetimeJulian(date.year, date.month + 1, 1)
+                else:
+                    end = cftime.DatetimeJulian(date.year + 1, 1, 1)
+                new_bounds[i, 0] = point.units.date2num(start)
+                new_bounds[i, 1] = point.units.date2num(end)
+            time.bounds = new_bounds
+        return cubes
+
+    def _fix_required(self, time):
+        if self.vardef.frequency != 'mon':
+            return False
+        if time.bounds[-1, 0] > time.points[-1]:
+            return True
+        if time.bounds[-1, 1] < time.points[-1]:
+            return True
+        return False
```

Comment:
```suggestion
        return (self.vardef.frequency == 'mon'
                and not (time.bounds[-1, 0] < time.points[-1] < time.bounds[-1, 1]))
```

Reply:
I also wonder where the frequency part of the condition is coming from. Does `sit` occur in another table? Is it ok in that table if the bounds are not surrounding the points?

Reply:
> Does sit occur in another table

It is also in day

> Is it ok in that table if the bounds are not surrounding the points?

Of course not, but I have not tested it and prefer to leave the fix out in case it is not needed